docs/sphinx/developers/matlab-dev.txt

```diff
@@ -19,7 +19,16 @@ MATLAB can be retrieved using::
 
 Default JVM settings can be increased by creating a :file:`java.opts` file in
 the startup directory and overriding the default memory settings. We recommend
-using ``-Xmx512m`` in your :file:`java.opts` file.
+using ``-Xmx512m`` in your :file:`java.opts` file. Calling::
+
+  bfCheckJavaMemory()
+
+will also throw a warning if the runtime memory is lower than the recommended
+value.
+
+If errors of type ``java.lang.OutOfMemoryError: Java heap space`` are thrown
+while using Bio-Formats, you may try to increase the default values of
+``-XX:MaxPermSize`` and ``-XX:PermSize`` via the :file:`java.opts` file.
```

Comment:
I do not think it is necessary since the JVM bundled with MATLAB is only Java 6 or 7 so far (http://en.wikipedia.org/wiki/MATLAB#Release_history)